tools/querytee/proxy_metrics.go

```diff
@@ -37,6 +38,11 @@ func NewProxyMetrics(registerer prometheus.Registerer) *ProxyMetrics {
 			Name:      "responses_compared_total",
 			Help:      "Total number of responses compared per route and backend name by result.",
 		}, []string{"backend", "route", "result", "issuer"}),
+		missingMetrics: promauto.With(registerer).NewHistogramVec(prometheus.HistogramOpts{
+			Namespace: "cortex_querytee",
+			Name:      "missing_metrics",
```

Comment:
I think using the term `metrics` here is confusing because loki doesn't really have that metrics per se... Can we rename this to `missing_metric_series` for clarity?